lib/absinthe/phase/schema/type_imports.ex

```diff
@@ -67,7 +67,12 @@ defmodule Absinthe.Phase.Schema.TypeImports do
   @dialyzer {:nowarn_function, [ensure_compiled: 1]}
   defp ensure_compiled(module) do
     if function_exported?(Code, :ensure_compiled!, 1) do
-      {:module, Code.ensure_compiled!(module)}
+      try do
+        {:module, Code.ensure_compiled!(module)}
```

Comment:
You should not rescue ensure_compiled!, as it will throw the compiler on a hoop. My suggestion is for you to have “ensure_compiled!” When you need the module and have a non raising one and call them accoedingly.